src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -34,7 +34,7 @@ public static class NodeConstants
         public const string YarnVersionCommand = "echo Using Yarn version: && yarn --version";
         public const string HugoVersionCommand = "echo Using Hugo version: && hugo version";
         public const string YarnPackageInstallCommand = "yarn install --prefer-offline";
-        public const string Yarn2PackageInstallCommand = "yarn plugin import workspace-tools && yarn workspaces focus --all";
+        public const string Yarn2PackageInstallCommand = "yarn workspaces focus --all";
```

Comment:
Doesn't this mean that users without workspaces will fail? You can call yarn install if the workspace focus subcommand is unavailable with:

```
yarn workspaces focus --all || yarn install
```

Reply:
yeah  good catch .. i will add it ...